### DISCUSSION TITLE
Link in WYSIWYG is not validated

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17077] [Archive]
### Describe the Bug

If you add a link to your WYSIWYG, it is not validated. I can put in other content than links.

### To Reproduce

Create a collection with a WYSIWYG field, create a new item and in the WYSIWYG add a link, where the link is `foobar`.

### Errors Shown

none :(

### What version of Directus are you using?

9.22.1

### What version of Node.js are you using?

Latest

### What database are you using?

MySQL

### What browser are you using?

Firefox

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
Anything can be put into an `<a>`'s `href`, so there's no hard requirement that the value is a valid domain / URL. Feel free to open a new [feature request](https://github.com/directus/directus/discussions/new/choose) so we can discuss adding a "validate links" option to the wysiwyg interface 🙂
